Testscripts/Linux/Sysbench.sh

```diff
@@ -0,0 +1,210 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+########################################################################
+#
+# Description:
+#       This script installs and runs Sysbench tests on a guest VM
+#
+#       Steps:
+#       1. Installs dependencies
+#       2. Compiles and installs sysbench
+#       3. Runs sysbench
+#       4. Collects results
+#
+#       No optional parameters needed
+#
+########################################################################
+CONSTANTS_FILE="constants.sh"
```

Comment:
these 2 variables should not be required

Reply:
Remove Constants variable